client/devdocs/style.scss

```diff
@@ -49,12 +49,6 @@ $devdocs-max-width: 720px;
 // Block and Component examples
 .main.devdocs.devdocs__blocks,
 .main.devdocs.devdocs__components,
-.main.devdocs.devdocs__gutenberg-blocks,
-.main.devdocs.devdocs__gutenberg-components {
-	// Make way for showing big components and multiple columns
-	max-width: 100%;
-}
```

Comment:
You've deleted too much stuff here!
We want to keep the `max-width: 100%` for `.devdocs__blocks` and `.devdocs__components`, not only because right now we are giving them the same rules as the next selector, but also because we want the docs to be nice and large! 😛 

Jokes aside, you can see that something is especially wrong in the Playground (`/devdocs/playground`), where the code editor is just too narrow to be usable.
Though, the error is visible in Blocks (`/devdocs/blocks`) and UI Components (`/devdocs/design`) as well.

Reply:
Ooops! Nice catch! Restored this in 2673416c2f122b1d5e91299d864ea48bf5ad2ed1. :)